### DISCUSSION TITLE
FormDialog如何手工触发“确定”和“取消”按钮的功能

### DISCUSSION THREAD

**Discussion by tsukasa521** [alibaba/formily#3003] [Q&A 2.x]
我使用的是@formily/element中FormDialog组件，我想自定义footer里的功能，但不知道如何触发默认的“确定”按钮和“取消”按钮的功能。求助，感谢

**Comment by janryWang** [comment]
为什么要手工触发？

**Reply by tsukasa521** [reply]
你好，在业务需求里dialog里是一个向导的功能。比如有三步，第一步只看到取消和下一步，第二步看到上一步和下一步，第三步看到上一步和确定。我想在footer里实现这种功能，手工来触发取消和确定。

**Reply by tsukasa521** [reply]
你好，有什么好的建议吗 @janryWang

**Reply by janryWang** [reply]
建议直接用Modal实现吧

**Reply by tsukasa521** [reply]
了解，我先这么来实现
